components/AddButton/index.js

```diff
@@ -0,0 +1,19 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCirclePlus } from "@fortawesome/free-solid-svg-icons";
+import styled from "styled-components";
+
+export default function AddButton({ handleClick }) {
+  return (
+    <AddProjectButton onClick={handleClick}>
+      <FontAwesomeIcon icon={faCirclePlus} />
+    </AddProjectButton>
+  );
+}
+
+const AddProjectButton = styled.div`
```

Comment:
For accessibility and screenreaders I would recommend to use a button tag, when you want to make a button, since we want to use semantic html.